QgisModelBaker/libili2db/ili2dbutils.py

```diff
@@ -44,14 +44,19 @@ def get_ili2db_bin(tool, stdout, stderr):
     db_simple_factory = DbSimpleFactory()
     db_factory = db_simple_factory.create_factory(tool)
 
-    ili_tool_version = db_factory.get_tool_version()
-    ili_tool_url = db_factory.get_tool_url()
+    ili_tool_version = db_factory.get_tool_version(db_ili_version)
+    ili_tool_url = db_factory.get_tool_url(db_ili_version)
 
     dir_path = os.path.dirname(os.path.realpath(__file__))
     ili2db_dir = '{}-{}'.format(tool_name, ili_tool_version)
 
-    ili2db_file = os.path.join(
-        dir_path, 'bin', ili2db_dir, '{}-{}.jar'.format(tool_name, ili_tool_version))
+    if db_ili_version == 3:
+         ili2db_file = os.path.join( dir_path, 'bin', ili2db_dir, '{tool}-{version}/{tool}.jar'
+                                     .format(tool=tool_name, version=ili_tool_version))
+    else:
+        ili2db_file = os.path.join(dir_path, 'bin', ili2db_dir, '{tool}-{version}.jar'
+                                   .format(tool=tool_name, version=ili_tool_version))
```

Comment:
This if/else will work if we do not update to ili2pg-3.12.2 or above. Perhaps this deserves a comment?

For instance, [ili2pg-3.12.2](http://www.eisenhutinformatik.ch/interlis/ili2pg/ili2pg-3.12.2.zip) does not hold the structure specified in line 54-55.

Reply:
Haven't checked that. Thanks.
I made it future proof by comparing exact version there. Please have quick look on it.

Reply:
I see this fine. 
Didn't know about the 'packaging' package. Always wrote a version comparing function by myself :)